airline-data/db_scripts/balance_audit_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE `balance_audit` (
```

Comment:
Ah forgot to mention, if it's new table, we dun need a script here . As a dev, i can just run airline-data/src/main/scala/com/patson/data/QuickCreateSchema.scala :)